cleanup-passes/travis.ts

```diff
@@ -18,25 +18,24 @@ import * as fs from 'fs';
 import * as yaml from 'js-yaml';
 import * as path from 'path';
 
-import {register} from '../cleanup-pass';
-import {ElementRepo} from '../element-repo';
-import {existsSync, makeCommit} from './util';
+import { register } from '../cleanup-pass';
+import { ElementRepo } from '../element-repo';
+import { existsSync, makeCommit } from './util';
```

Comment:
`npm run format` if that command exists plz

otherwise don't worry about it

Reply:
Ah this was VS Code's doing.

Reply:
No `format` script. Should I port over the `clang-format` setup from `polymer-cli`?